### PR TITLE
fix humphd/brackets#117 - Indentation looks incorrect on the <tr> tag

### PR DESCRIPTION
**Pull request by AJDBenner**
This PR fixes https://github.com/humphd/brackets/issues/117. It also changes the spaceUnits and tabSize to both be 2 which keeps the auto indenting and tabs consistent with each other.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -161,6 +161,11 @@ define(function (require, exports, module) {
         //Below are methods to change the preferences of breackets, more available at:
```
Comment (sedge) [inline main.js]: Only cuz I just spotted it, but can you fix this typo?

Comment (AJDBenner) [inline main.js]: Good catch.

**Comment by sedge** [comment]
r+, nice one.

**Comment by humphd** [comment]
We'll need to land this in bramble, but let's hold off until @sedge says it's green.

**Pull request merged**